### DISCUSSION TITLE
vampires may also like bright light theme

### DISCUSSION THREAD

**Discussion by avelino** [dracula/dracula-theme#508] [Q&A] [answered]
> "Are you going to create a light color scheme?"
> Nope. Dracula can't stand the light.

The dracula theme was born with this premise, but many years have passed since its beginning, isn't it time to review this point?
I use the dracula (drak) theme daily, but there is a day that I need to keep a light theme, do I only give bipolar (there is a day that I want dark and others light)?

ref: https://marketplace.visualstudio.com/items?itemName=ashrafhadden.dracula-dot-min

**Comment by benknoble** [comment]
Idk if this is still the current answer, but at one time it was simply too much to do with what was available: https://github.com/dracula/dracula-theme/issues/209

**Reply by avelino** [reply]
From 2016 to 2021 it has been 5 years and a lot has changed in the project (we support a lot of software), isn't it time to relive this subject?

**Reply by benknoble** [reply]
I'm not stating one way or another; in fact, I would argue it's reasonable to have the discussion again now. I just want to point us to the old discussion (and prepare us for "the answer hasn't changed").

**Comment by zenorocha** [comment]
That's a very interesting topic and, to be completely honest, I have some mixed feelings about it.

**The good side**
- I feel like light themes are super useful and relevant. They were important in the past and will continue to be in the future.
- Dracula has a strong community and a pretty well-organized structure. If we decide to move in this direction I'm positive that we can create the best light theme out there.

**The bad side**
- The amount of work that it takes to change and re-publish everything that we currently have is massive. Yes, there is some automation that could be done, but it's not as simple as it seems.
- I personally never used a light theme in 13+ years of coding, so it's really hard for me to build something that I'm not personally interested in. Again, I understand how this can be useful for others, and that's a pretty good motivation by itself. However, when I compare this with all the plans I still have for Dracula, I feel like we should continue to invest in creating the most universal dark mode ever.

**Reply by zenorocha** [reply, marked as answer]
After writing this years ago, I've changed my mind and decided to come back here to explain why.

Even though I'm still not a huge light theme user, the Dracula project grew beyond my personal preferences, and we must respect the needs of other people.

A light theme goes beyond personal style. It is crucial to improve accessibility and usability in brighter environments, ensuring that people with visual impairments can comfortably use a theme.

That's why we're releasing a light theme today called [Alucard](https://draculatheme.com/pro).

**Comment by CircleCode** [comment]
I may be resurrecting an old topic (I may also have said that I am digging the topic out), But what attracts me in Dracula is its universality: you keep homogeneous colors for all the tools you use, and I am even keen to pay for it.
However, I am currently personally favoring  light themes over dark ones.
Given the amount of work, I understand that even if you were about to create a light variant, there would need some time. In the meantime, are you aware of a universal light theme until you create your own?
